packages/integration-tests/src/tests/api/hook/hook.trigger.test.ts

```diff
@@ -226,7 +228,8 @@ describe('trigger hooks', () => {
     // Wait for the hook to be trigged
     await waitFor(1000);
 
-    const logs = await getLogs(
+    const logs = await getWebhookRecentLogs(
+      resetPasswordHook.id,
```

Comment:
Why add this `id`?

Reply:
`getWebhookRecentLogs` is a helper function for requesting a specific hook's (characterized by hook `id` in this case) webhook logs.